adapter/internal/oasparser/envoyconf/http_filters.go

```diff
@@ -45,6 +46,7 @@ import (
 func getHTTPFilters() []*hcmv3.HttpFilter {
 	extAauth := getExtAuthzHTTPFilter()
 	router := getRouterHTTPFilter()
+	lua := getLuaFilter()
```

Comment:
We only need to set lua filter when there is a mediation service. Otherwise, we should not have any lua filter code in the envoy config.

Reply:
Based on your code, you add the filter globally and disable it per route?

Reply:
yes, we have to add the lua filter under listener's HTTP filters. Then we can enable/disable the Lua filter per route. By default, the filter is disabled in per route configurations. if a mediation service exists(aka if the swagger extension exists ) for the resource/API, it will be enabled.

Reply:
Ack